cumulusci/robotframework/pageobjects/PageObjects.py

```diff
@@ -1,22 +1,36 @@
 from robot.api import logger
-from robot.libraries.BuiltIn import BuiltIn
-from .baseobjects import BasePage
+from robot.libraries.BuiltIn import BuiltIn, RobotNotRunningError
+from cumulusci.robotframework.pageobjects.baseobjects import BasePage
 import inspect
 import robot.utils
 import os
 import sys
 
 
+def get_keyword_names(obj):
+    """Returns a list of method names for the given object
+
+    This excludes methods that begin with an underscore, and
+    also excludes the special method `get_keyword_names`.
+    """
+    names = [
+        member[0]
+        for member in inspect.getmembers(obj, inspect.isroutine)
+        if (not member[0].startswith("_")) and member[0] != "get_keyword_names"
+    ]
```

Comment:
This is definitely a minor stylistic suggestion, but I would do

    names = [name for name, member in inspect.getmembers(obj, inspect.isroutine) if not name.startswith("_") and name != "get_keyword_names"]

I try to avoid using numeric indexes when it's easy to use a more meaningful name instead.